dom-manipulation/main.js

```diff
@@ -0,0 +1,27 @@
+var numberClicked = 0;
+var $hotButton = document.querySelector('.hot-button');
+var $clickCount = document.querySelector('.click-count');
+
+function numberClickedfunction() {
+  numberClicked++;
+}
+$hotButton.addEventListener('click', numberClickedfunction);
```

Comment:
You should include this in the other event handler rather than having two separate event handlers for the same event on the same element.